lib/deployment_runner.py

```diff
@@ -117,8 +117,12 @@ def rhv_configuration(self, page):
 
         # Custom data center & cluster name in RHVSH disabled for QCI 1.0
         # See https://bugzilla.redhat.com/show_bug.cgi?id=1367777
-        if self.rhv.rhv_setup_type != 'selfhost':
+        # Temporarily disabled for RHV in QCI 1.1
+        # See https://bugzilla.redhat.com/show_bug.cgi?id=1413194
```

Comment:
You know even though you coded this to deal with the temporary change in behavior, the way you coded it can stay that way forever (or as long as those fields exist).    That is unless they were not supposed to be disabled in a certain context at some point.  I guess there is nothing to act on, just thinking out loud I guess.